src/main/java/io/cryostat/rules/MatchExpressionEvaluator.java

```diff
@@ -128,6 +130,15 @@ private void invalidate(String matchExpression) {
         }
     }
 
+    public void evaluates(String matchExpression) throws ScriptException {
+        try {
+            ServiceRef dummyRef = new ServiceRef("jvmId", new URI("file:///foo/bar"), "alias");
```

Comment:
Since the URI string is hardcoded here and known-safe, you could use `URI.create()` instead and avoid the `try/catch`.